src/OneSignalMessage.php

```diff
@@ -45,212 +33,131 @@ public static function create($body = '')
      */
     public function __construct($body = '')
     {
-        $this->body = $body;
+        $this->setBody($body);
     }
 
     /**
      * Set the message body.
      *
-     * @param string $value
+     * @param mixed $value
      *
+     * @deprecated use setBody instead
+     *             
      * @return $this
      */
     public function body($value)
```

Comment:
I wonder if it would be good to move these deprecated methods to `Traits\Deprecated` to clean things up?